tests/chainer_tests/functions_tests/normalization_tests/test_batch_normalization.py

```diff
@@ -231,6 +245,15 @@ def f(*inputs):
                 **self.check_backward_options)
 
     def test_backward(self, backend_config):
+        if isinstance(backend_config.device, chainer.backend.Intel64Device):
```

Comment:
Functions should work regardless of backends. Instead of modifying tests, please fix the code.
I think you should fix `_BNMode.can_use_ideep()` in `batch_normalization.py`.